From 24ab1642e98d86044fb964a1765eaa43fc4f187f Mon Sep 17 00:00:00 2001
From: Marcin Bis <marcin@bis-linux.com>
Date: Mon, 29 Apr 2019 18:40:38 +0000
Subject: [PATCH] Whitelist kernel v4.20.12 as we explicite patch it

---
 blacklist/kprobes.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/blacklist/kprobes.h b/blacklist/kprobes.h
index 6c1d7a5..ed7c118 100644
--- a/blacklist/kprobes.h
+++ b/blacklist/kprobes.h
@@ -12,7 +12,7 @@
 
 #include <lttng-kernel-version.h>
 
-#if LTTNG_KERNEL_RANGE(4,20,0, 4,20,13) \
+#if LTTNG_KERNEL_RANGE(4,20,0, 4,20,11) \
 	|| LTTNG_KERNEL_RANGE(4,19,9, 4,19,26) \
 	|| LTTNG_KERNEL_RANGE(4,14,87, 4,14,104)
 # if defined(CONFIG_ARM) && defined(CONFIG_OPTPROBES)
-- 
2.17.1

